packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -28,11 +28,11 @@ import { timeoutAwait } from "./timeoutUtils";
 
 const summarizerClientType = "summarizer";
 
-async function createSummarizerCore(
+export async function createSummarizerCore(
```

Comment:
This is not supposed to be exported. The other createSummarizer APIs should be used instead. If you are adding something generic to the API, you should add it to the other APIs instead.

Reply:
Unexported